docs/whats-new.asciidoc

```diff
@@ -1,17 +1,95 @@
 [[whats-new]]
 [chapter]
-= What's new
+= What's new in 7.16
 
-Here are the highlights of what’s new and improved in {es-sec}!
+Here are the highlights of what’s new and improved in {elastic-sec}!
 
 For detailed information about this release, see the <<release-notes, Release notes>>.
 
 Other versions: {security-guide-all}/7.15/whats-new.html[7.15] | {security-guide-all}/7.14/whats-new.html[7.14] | {security-guide-all}/7.13/whats-new.html[7.13] | {security-guide-all}/7.12/whats-new.html[7.12] | {security-guide-all}/7.11/whats-new.html[7.11] | {security-guide-all}/7.10/whats-new.html[7.10] |
 {security-guide-all}/7.9/whats-new.html[7.9]
-////
 
 [discrete]
-[[sec-7.15-release]]
-== 7.15 release
+[[sec-features-7.16]]
+=== New features
 
-////
+*Memory threat protection added for Linux and macOS hosts*
+
+<<memory-protection, Memory threat protection>>, which detects and stops many of the techniques used for process injection, is now added to Linux and macOS hosts. It ships with preventions for shellcode (macOS and Linux), reflective ELF injection (Linux), and in-memory Mach-O execution (macOS), plus the preventions for Windows systems that were introduced in {stack} version 7.15. Memory threat protection is available for customers with a https://www.elastic.co/pricing[Platinum or Enterprise license].
+
+*Expanded malicious behavior protections*
```

Comment:
maybe just "protection" instead of "protectionS", for consistency with the previous heading